test/interval_tests/numeric.jl

```diff
@@ -434,3 +434,30 @@ end
     @test nthroot(Interval{BigFloat}(-81, -16), -4) == ∅
     @test nthroot(Interval{BigFloat}(-81, -16), 1) == Interval{BigFloat}(-81, -16)
 end
+
+# approximation used for testing (not to rely on ≈ for intervals)
+# ⪆(x, y) = (x ≈ y) && (x ⊇ y)
+⪆(x::Interval, y::Interval) = x.lo ≈ y.lo && x.hi ≈ y.hi && x ⊇ y
```

Comment:
I think `x ≈ y` works for intervals (instead of `x.lo ≈ y.lo && x.hi ≈ y.hi`)...

Reply:
Unfortunately not
``` julia
julia> 1.00000001 ≈ 1.0
true

julia> 1.0000000001..2.0 ≈ 1.0..2.0
false

```

Reply:
Good point!

Reply:
Ref #479 #480

Reply:
Also, the returned interval should be at least as wide as the correct result.